docs/en/sql-reference/functions/date-time-functions.md

```diff
@@ -1274,7 +1274,7 @@ Alias: `SUBDATE`
 **See Also**
 - [date_sub](#date_sub)
 
-## now
+## now {#date_time_functions-now}
```

Comment:
Do we need to add `date_time_functions-` part? Currently, it works with just `#now` anchor, and it looks just a little bit better.

Reply:
I agree, the prefix is not very pretty. On the other hand, a prefix is convention in other markdown files.

In theory, a prefix is superfluous, perhaps I can make another PR which removes it from everywhere.

Reply:
> a prefix is convention in other markdown files.

Yep, at some point we changed how the anchor links are generated, and currently almost none of them work. But I think it is better to fix it by changing the anchors in other files rather than making them ugly again

Reply:
Okay, done.